stetho/src/main/java/com/facebook/stetho/inspector/network/ResponseBodyFileManager.java

```diff
@@ -43,11 +43,10 @@
   private static final int PRETTY_PRINT_TIMEOUT_SEC = 10;
 
   private final Context mContext;
-
   private final Map<String, AsyncPrettyPrinter> mRequestIdMap = Collections.synchronizedMap(
       new HashMap<String, AsyncPrettyPrinter>());
-  private final ExecutorService sExecutorService =
-      AsyncPrettyPrinterExecutorHolder.sExecutorService;
+
+  private ExecutorService sExecutorService;
```

Comment:
Same here, no need to keep a static reference to something you can just call a getter for. Especially when it is to a mutable object